Exception thrown if fluo yarn commands run on empty Zookeeper

This exception is thrown every time the command `fluo-dev deploy` is run with fresh Zookeeper.

No root node '/' exists in ZK which causes exception.  Commands should check if root node exists before reading ZK.
```
Exception in thread "YarnTwillRunnerService STARTING" java.lang.RuntimeException: java.util.concurrent.ExecutionException: org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /
	at com.google.common.base.Throwables.propagate(Throwables.java:160)
	at com.google.common.util.concurrent.AbstractIdleService$1$1.run(AbstractIdleService.java:47)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.util.concurrent.ExecutionException: org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /
	at com.google.common.util.concurrent.AbstractFuture$Sync.getValue(AbstractFuture.java:294)
	at com.google.common.util.concurrent.AbstractFuture$Sync.get(AbstractFuture.java:281)
	at com.google.common.util.concurrent.AbstractFuture.get(AbstractFuture.java:116)
	at org.apache.twill.yarn.YarnTwillRunnerService.startUp(YarnTwillRunnerService.java:304)
	at com.google.common.util.concurrent.AbstractIdleService$1$1.run(AbstractIdleService.java:43)
	... 1 more

```

--- a/modules/cluster/src/main/java/io/fluo/cluster/yarn/YarnAdmin.java
+++ b/modules/cluster/src/main/java/io/fluo/cluster/yarn/YarnAdmin.java
@@ -28,6 +28,7 @@ import io.fluo.api.config.FluoConfiguration;
 import io.fluo.cluster.FluoOracleMain;
 import io.fluo.cluster.FluoWorkerMain;
 import io.fluo.cluster.util.LogbackUtil;
+import io.fluo.core.client.FluoAdminImpl;
 import io.fluo.core.util.CuratorUtil;
 import org.apache.commons.configuration.ConfigurationException;
 import org.apache.curator.framework.CuratorFramework;
@@ -258,6 +259,11 @@ public class YarnAdmin {
 
     File configFile = new File(options.getFluoConf() + "/fluo.properties");
     config = new FluoConfiguration(configFile);
+    
+    if (!FluoAdminImpl.zookeeperInitialized(config)) {
+      System.out.println("ERROR - Fluo has not been initialized yet in Zookeeper at " + config.getZookeepers());
+      System.exit(-1);
+    }
 
     try {
       curator = CuratorUtil.newFluoCurator(config);

